app/views/admin/products_v3/_table.html.haml

```diff
@@ -41,7 +41,7 @@
         %td.align-left
           .line-clamp-1= product.taxons.map(&:name).join(', ')
         %td.align-left
-          .line-clamp-1= product.tax_category&.name || "None" # TODO: convert to dropdown, else translate hardcoded string.
+          .line-clamp-1= product.tax_category_id.present? && product.tax_category&.name  || "None" # TODO: convert to dropdown, else translate hardcoded string.
```

Comment:
You don't need `present?` on an id. It's either present or nil. And applying Rachel's comment:
```suggestion
          .line-clamp-1= product.tax_category_id && product.tax_category&.name  || ""
```